Builder_/Program.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace builder_
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            // Client Code
+            var whitePizzaBuilder = new WhitePizzaBuilder();
```

Comment:
It would be better if you used polymorphism to define builders, without creating new builder variable for each one.